driver/src/main/scala/api/DBMetaCommands.scala

```diff
@@ -127,4 +128,11 @@ trait DBMetaCommands { self: DB =>
       self, command, ReadPreference.primary
     ).map(_ => {})
   }
+
+  /**
+   * Returns 1.0 when the server is responding to commands.
```

Comment:
I added an explanation of the command and provided control of readPreference to caller. The default is `Nearest` because the successful response in mongo does not indicate if a write-lock is present.